ggml/src/ggml-rpc/ggml-rpc.cpp

```diff
@@ -2053,6 +2054,9 @@ ggml_backend_reg_t ggml_backend_rpc_reg(void) {
 
 static uint32_t ggml_backend_rpc_get_device_count(const char * endpoint) {
     auto sock = get_socket(endpoint);
+    if (sock == nullptr) {
+        return 0;
```

Comment:
If the endpoint is in the correct format but we cannot connect we should also log an error here:
```
GML_LOG_ERROR("Failed to connect to %s\n", endpoint);
```